premailer/merge_style.py

```diff
@@ -10,11 +10,9 @@ def csstext_to_pairs(csstext):
     # The lock is required to avoid ``cssutils`` concurrency
     # issues documented in issue #65
     with csstext_to_pairs._lock:
-        parsed = cssutils.css.CSSVariablesDeclaration(csstext)
-        return [
-            (key.strip(), parsed.getVariableValue(key).strip())
-            for key in sorted(parsed)
-        ]
+        return [(prop.name, prop.propertyValue.cssText)
+                for prop in cssutils.parseStyle(csstext)]
+
```

Comment:
This is quite a big change. Before the name and the value used to be whitespace stripped. Also, before it used to sort the parsed CSS. 

Can we perhaps have a new test to this PR that demonstrates what this change above achieves that we couldn't do before?